daemon/gluster-blockd.c

```diff
@@ -262,13 +265,70 @@ glusterBlockDParseArgs(int count, char **options)
   return 0;
 }
 
+
+static void
+gbMinKernelVersionCheck(void)
+{
+  struct utsname verStr = {'\0', };
+  size_t vNum[VERNUM_BUFLEN] = {0, };
+  int i = 0;
+  char *tptr;
+
+
+  if (uname(&verStr) != 0) {
+    LOG("mgmt", GB_LOG_ERROR, "uname() failed: %s", strerror(errno));
+    exit(EXIT_FAILURE);
+  }
+
+  tptr = verStr.release;
+  while (*tptr) {
+    if (isdigit(*tptr)) {
+      vNum[i] = strtol(tptr, &tptr, 10);
+      i++;
+    } else if (isalpha(*tptr)) {
+      break;
+    } else {
+      tptr++;
+    }
+
+    if (i >= VERNUM_BUFLEN) {
+      break;
+    }
+  }
+
+  /* Minimum recommended kernel version: 4.13.13-300.fc27.x86_64 */
```

Comment:
Why only the fedora kernel version ? Then the upstream code won't work in the RHEL and CentOS.